bindata/v4.0.0/controller/role.yaml

```diff
@@ -37,3 +37,5 @@ rules:
   - get
   - list
   - watch
+  # Support scaling the unified deployment to zero on downgrade to multiple independent deployments
+  - update
```

Comment:
This should be `delete` now

Reply:
Good catch, and prompt to ensure deletion of the other resources (and indicate that this permission can be revoked in 4.5).

Reply:
Updated, PTAL